monitoring/README.md

```diff
@@ -0,0 +1,234 @@
+# Monitoring Setup for Agentic-on-EKS
```

Comment:
Can you add a section named `Observability Setup` in the main README.md(https://github.com/agentic-community/agentic-on-eks/blob/main/README.md) and add a link to monitoring/README.md?